### ISSUE TITLE
tasks alternative to kanban

### ISSUE THREAD

**Issue by aegis301** [helpwave/helpwave#129]

**Comment by 00100200345** [comment]
As medical professionals we should ask ourselves if a specific way of doing things (in this case: handling tasks) is 
a. a proper way to represent something digitally as it is done at the moment or 
b. improving a specific process. 

I am not sure if the "kanban" style is either a or b when it comes to handling tasks. 

The few cases where "in progress" is useful category are not worth the design choices to be made for this style. The kanban board as it is now takes up a lot of space and is cluttered. The transition from this to something like an overview of what was done during a patient's stay is less streamlined than for example a list of current todo and a list of finished tasks, both date/time-related. Options such as applying filters are also likely to be less simple (aesthetically). 

Lastly, the kanban style is something not commonly used in hospitals (it's a very digital solution). In order to ease the transition from using pen and paper to something like tasks, I think, we should go for a solution aiming for "a" (see above"), at least as long as we don't have a clear case of a "b" solution.

**Comment by aegis301** [comment]
I agree with the point that the kanban board, especially the "in progress" status might be overkill. I absolutely agree with the filtering point. Kanban boards make that more difficult in general. However I think it is crucial to indicate the current status of a task. E.g. is the registration for the tumor board already in progress by a colleague? The last thing we want is for people to do things twice or to have to call a colleague to ask if they're already working on it. I don't think we have to propose a solution (color coding?) however I think we should keep in mind that this is crucial to the workflow.

**Comment by 00100200345** [comment]
That's true, being able to see a status of a tasks is very useful. Given that at the moment, we use a circle to indicate whether a tasks is open, done or in progress (when it has subtasks), would something like a combination of having this kind of indicator and the use a simple "in progress/wird bearbeitet" phrase not be sufficient (a text field with background coloring e.g., similar to the circle). I know, we don't need/should propose solutions, I just wanted to throw it out there. 
In short: I think the status "in progress" can be indicated when showing (a list?) of tasks that are not finished. I think that should suffice the need of being able to see if someone else is already handling that tasks? What do you think?

**Comment by sophiafe** [comment]
I agree. The status "in progress" might be relevant for a task with multiple subtasks, when some of the subtasks have already been completed, i.e. the task has been started. Like Ludwig said, that could be indicated in a different way than a kanban board style. 

A exemplary use case for "in progress" is if a patient had blood cultures drawn (this could be part of a task template called "sepsis diagnostics") and the medical team has to remember to check whether the microbiology results came back. This could also be solved as an extra task/subtask "check microbiology results". Therefore, the "in progress" status is not essential in this case.

Especially with the properties added to the patient view, I feel like we should try to use the space more efficiently. I think as soon as many tasks and properties are added, the user won't have a good overview.

**Comment by aegis301** [comment]
MVP:
- [ ] change status to: to do, done -> "archive view" necessary where tasks with status done can be viewed
- [ ] additional abilty to assign tags to tasks (to-be-reviewed, to-be-discussed....)

non-MVP:
- [ ] filter functions are necessary, especially for handovers where tasks of the day
